components/sidebar/sidebar_channel/sidebar_channel_link/index.ts

```diff
@@ -44,13 +42,9 @@ function makeMapStateToProps() {
         const tutorialStep = getInt(state, TutorialTourName.ONBOARDING_TUTORIAL_STEP, currentUserId, 0);
         const triggerStep = getInt(state, OnboardingTaskCategory, OnboardingTasksName.CHANNELS_TOUR, FINISHED);
         const channelTourTriggered = triggerStep === GenericTaskSteps.STARTED;
-        const nextStep = showNextSteps(state);
-        const onboardingHidden = isOnboardingHidden(state);
         const isOnboardingFlowEnabled = config.EnableOnboardingFlow;
-        const isUserFirstAdmin = isFirstAdmin(state);
         const showChannelsTour = enableTutorial && tutorialStep === OnboardingTourSteps.CHANNELS_AND_DIRECT_MESSAGES;
-        const showChannelsTutorialStep = showChannelsTour && ((channelTourTriggered && isUserFirstAdmin) || (isOnboardingFlowEnabled === 'true' && !(nextStep || onboardingHidden) && !isUserFirstAdmin) || (isOnboardingFlowEnabled !== 'true' && !isUserFirstAdmin));
-
+        const showChannelsTutorialStep = showChannelsTour && channelTourTriggered && isOnboardingFlowEnabled === 'true';
```

Comment:
I haven't dug in, but it looks like this logic is not enough. I'm guessing there's something from the first admin onboarding that doesn't get set for other users? When I create a new regular user on the spinwick, I'm not getting the task list showing.

Reply:
let me check that